src/CoreBundle/private/library/classes/TCMSURLHistory.class.php

```diff
@@ -21,7 +21,17 @@ class TCMSURLHistory
      */
     public $aHistory = array();
 
-    public $index = 0;
+    /**
+     * @deprecated since 6.3.0 - use getHistoryIndex() instead
```

Comment:
A docblock must belong to a code entity, so we can't use it here. Please trigger a deprecation warning instead (see `TModelBase::__get` for an example).

Reply:
done